presto-tests/src/test/java/com/facebook/presto/tests/TestSqlFunctions.java

```diff
@@ -86,9 +86,7 @@ public void testCreateFunctionInvalidFunctionName()
     @Test
     public void testCreateFunctionInvalidSemantics()
     {
-        assertQueryFails(
-                "CREATE FUNCTION testing.common.tan (x int) RETURNS varchar COMMENT 'tangent trigonometric function' RETURN sin(x) / cos(x)",
-                "Function implementation type 'double' does not match declared return type 'varchar'");
+        assertQuerySucceeds("CREATE FUNCTION testing.common.tan (x int) RETURNS varchar COMMENT 'tangent trigonometric function' RETURN sin(x) / cos(x)");
```

Comment:
As we discussed offline. We want to test that those functions can not only be created, but also be executed. Please add test cases that invokes this created function.

Reply:
Fixed, added unit tests to test execution.